src/core/python/transform_v.cpp

```diff
@@ -86,106 +86,108 @@ void bind_transform3(nb::module_ &m, const char *name) {
 template <typename Float, typename Spectrum>
 void bind_transform4(nb::module_ &m, const char *name) {
     MI_IMPORT_CORE_TYPES()
-    using Ray3f = Ray<Point<Float, 3>, Spectrum>;
+    using _Point4f = Point<Float, 4>;
+    using _Transform4f = Transform<_Point4f>;
+    using Ray3f = Ray<Point3f, Spectrum>;
```

Comment:
Nope, I think I was originally having issues with `Ray3d` types being resolved with the stubs and thought this might fix it. But with the latest changes to the stubs this isn't required and seems to work fine so I've removed it.